ivy/__init__.py

```diff
@@ -355,6 +355,120 @@ def __dir__(self):
     def shape(self):
         return self._shape
 
+    @property
+    def value(self):
+        return self._value
+
+    def concatenate(self, other):
+        if self._shape is None or other.dims is None:
+            raise ValueError("Unknown Shape")
+        else:
+            return Shape(self.dims + other.dims)
+
+    def index(self, index):
+        assert isinstance(self._shape, Shape)
+        if self._shape.rank is None:
+            return Shape(None)
+        else:
+            return self._shape[index]
+
+    @property
+    def shape(self):
+        return self._shap
```

Comment:
Hey @soma2000-lang i think the shape property already exists above, you can remove this